### PR TITLE
Profile Page: Filling out Tab Content

### PR DESCRIPTION
**Pull request by axelauda**
---
name: "Filling out Tab Content"
about: "This will implement the layout and content of the tabs on the Profile, and "
title: "feat: "
labels: "feature-request, not-reviewed"
assignees: @axelauda
---

# Description
- Added a description box to the About Tab
  - User can swap between editing and viewing description by clicking the edit button at the top right
- Added a two column grid to the Friends Tab
  - Displays friends based off an array of friend info (profile picture and username)
- Fill Friends Tab with friend placeholders to showcase what it would look like
- Added a three column grid to the Trips Tab
  - Use the same trip cards used for My Trips page
- All tabs with content that extends the height of the profile becomes scrollable

Fixes # (issue)
- Fixes issue #48.

# Add an image of the change (if applicable)
image
image
image

# Type of change

- [ ] Bug fix 
- [x] New feature 
- [ ] Breaking change 
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Niblick1020 — APPROVED** [review]
After reviewing pull request #50, I'm confident these changes will significantly improve the Profile Page's functionality and readability. The updates, including streamlined user interactions and code cleanups, align well with our project goals. I support merging this PR into the master branch as it positively impacts our app's development and user experience.

**Review by VyProductions — CHANGES_REQUESTED** [review]
While the content of the profile changes look great, I can't approve this pull request because you are removing the `ISSUE_TEMPLATE` content and the `pull_request_template.md` file from the master branch. Please update your commit history to exclude changes to those files.

After this issue is resolved I will approve the pull request.

**Review by VyProductions — APPROVED** [review]
Confirmed that template files have been discarded from commit history. Thank you.

**Pull request merged**
